.github/actions/setup-macos/action.yml

```diff
@@ -22,17 +22,14 @@ outputs:
 runs:
   using: "composite"
   steps:
-    # github actions overwrites brew's python. Force it to reassert itself, by running in a separate step.
-    # Borrowed from https://github.com/mesonbuild/meson/blob/aab2533ab4f7f4c16991620b400d71782f89be1c/.github/workflows/macos.yml#L87-L92
-    # until https://github.com/actions/setup-python/issues/577 is fixed.
-    - name: unbreak python in github actions
+    - name: Force update all brew python
       run: |
-        find /usr/local/bin -lname '*/Library/Frameworks/Python.framework/2to3*' \
-          -o -lname '*/Library/Frameworks/Python.framework/idle3*' \
-          -o -lname '*/Library/Frameworks/Python.framework/pydoc3*' \
-          -o -lname '*/Library/Frameworks/Python.framework/python3*' -delete
-        sudo rm -rf /Library/Frameworks/Python.framework/
-        brew install --force python3 && brew unlink python3 && brew unlink python3 && brew link --overwrite python3
+        brew update
+        # GHA messes with links https://github.com/actions/setup-python/issues/577 so we just force replace their installs' with brew's
```

Comment:
Nit: Delete the `'` after "installs"

Reply:
Hmm... Perhaps it should be changed to `their install's`?

Reply:
Or perhaps it is short for `their installations' links`? I guess we can leave it unchanged.